sprimber-engine/src/main/java/com/griddynamics/qa/sprimber/engine/executor/ErrorMapper.java

```diff
@@ -50,6 +52,12 @@ public ErrorMapper(List<String> skippedExceptionNames,
         this.pendingExceptionAnnotations = pendingExceptionAnnotations;
     }
 
+    @PostConstruct
+    private void sortCollections() {
+        Collections.sort(skippedExceptionNames);
```

Comment:
How will it work with list changes on client side? For example if failedExceptionNames is modified on client side via postConstruct